src/jit/target.h

```diff
@@ -484,9 +484,15 @@ typedef unsigned short regPairNoSmall; // arm: need 12 bits
   #define MIN_ARG_AREA_FOR_CALL    0       // Minimum required outgoing argument space for a call.
 
   #define CODE_ALIGN               1       // code alignment requirement
+#if !defined(FEATURE_PAL)
   #define STACK_ALIGN              4       // stack alignment requirement
```

Comment:
As I understand, it is not a part of standard ABI.  It is just a requirement of GCC (> 4.5) and corresponding Clang.